nvflare/private/fed/server/server_commands.py

```diff
@@ -180,8 +179,9 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         shareable.set_header(key=FLContextKey.TASK_ID, value=task_id)
 
         shareable.set_header(key=ServerCommandKey.TASK_NAME, value=taskname)
-        shared_fl_ctx = FLContext()
-        shared_fl_ctx.set_public_props(copy.deepcopy(get_serializable_data(fl_ctx).get_all_public_props()))
+
+        # TODO
```

Comment:
@IsaacYangSLA what is the TODO here about?

Reply:
I think this PR just addressed this "TODO".

Reply:
Yan is right.  I put that TODO when searching for places to update as reminders.